bdk-swift/Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "bdk-swift",
+    platforms: [
+        .macOS(.v12),
+        .iOS(.v15)
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "BitcoinDevKit",
+            targets: ["bdkFFI", "BitcoinDevKit"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+//        .binaryTarget(
+//            name: "bdkFFI",
+//            url: "https://github.com/bitcoindevkit/bdk-swift/releases/download/0.3.0/bdkFFI.xcframework.zip",
+//            checksum: "7d4a2fdeb03fb3eff107e45ee3148dd9b67966406c82d6e3c19f653c27180cfd"),
```

Comment:
Is this the part that gets uncommented/fixed with the new path when running the final publishing workflow? I remember you telling me about this but not sure what it does anymore...

Reply:
yes this part get's added/un-commented and the other `.binaryTarget` that points to the local path get's commented out.